bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/widgets/Combo.java

```diff
@@ -1146,15 +1146,16 @@ void menuDidClose(long id, long sel, long menu) {
 void mouseDown(long id, long sel, long theEvent) {
 	// If this is a combo box with an editor field and the control is disposed
 	// while the view's cell editor is open we crash while tearing down the
-	// popup window. Fix is to retain the view before letting Cocoa track
-	// the mouse events.
+	// popup window. Fix is to retain the display and view before letting Cocoa
+	// track the mouse events.
+
+	final Display display = this.display;
+	final NSView view = this.view;
 
 	display.sendPreExternalEventDispatchEvent();
-	// 'view' will be cleared if disposed during the mouseDown so cache it.
```

Comment:
Let's retain the comment and variable name as they don't need any changes.

Reply:
I would say that this is a matter of taste. And these a better left to author, if there's no obvious harm.

Reply:
Not sure if reply is expected here. I'll resolve the conversation to merge PR.